docs/source/tensorbay_cli/tbrn.rst

```diff
@@ -1,3 +1,82 @@
 #########################
  TensorBay Resource Name
 #########################
+
+TensorBay Resource Name(TBRN) uniquely defines the different section stored in TensorBay.
+TBRN begins with ``tb:``. The following is the general format for TBRN:
+
+.. code:: console
+
+   tb:[dataset_name]:[segment_name]://[remote_path]
+
+It has 3 types:
+
+| 1. `TBRNType.DATASET`
+
+.. code:: console
+
+    tb:VOC2012
+
+The TBRN means the dataset "VOC2012".
+
+| 2. `TBRNType.SEGMENT`
+
+.. code:: console
+
+    tb:VOC2010:train
+
+The TBRN  means the "train" segment of dataset "VOC2012".
+
+| 3. `TBRNType.NORMAL_FILE`
+
+.. code:: console
+
+    tb:VOC2012:train://2012_004330.jpg
+
+The TBRN  means the file "2012_004330.jpg" of the "train" segment in normal dataset "VOC2012".
+
+
+Moreover, the file folder is supported by adding "/" at the end of remote_path.
+
+.. code:: console
+
+    tb:VOC2012:train://test1/
+
+The TBRN  means all files in the folder of "test1" of the "train" segment in normal dataset "VOC2012".
+
+
+************************
+ TBRN With Version Info
+************************
+
+The version information can also be included in the TBRN.
+See more details in :ref:`features/version_control:Version Control`.
+The following is the TBRN with version information:
+
+| 1. Commit:
+
+.. code:: console
+
+   tb:[dataset_name]@[revision]:[segment_name]://[remote_path]
+
+.. code:: console
+
+    tb:VOC2010@main
+
+The TBRN  means the dataset "VOC2012" is on the status of :ref:`reference/glossary:commit` and
+the :ref:`reference/glossary:revision` is "main". Actually, if neither revision
+nor draft number is given, the default revision: "default branch" is stored in TBRN.
+
+| 2. Draft:
+
+.. code:: console
+
+   tb:[dataset_name]#[draft_number]:[segment_name]://[remote_path]
+
+
+.. code:: console
+
+    tb:VOC2012#1
+
+The TBRN means the dataset "VOC2012" is on the status of :ref:`reference/glossary:draft` and the draft number is 1.
+
```

Comment:
Note that if neither revision nor draft number is given, a TBRN will refer to the default branch.